test/mocks/event.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
is it worthwhile to split some of these into consts?

Reply:
You've convinced me.  I've split the mocks folder into `events` and `values`.  Where events objects are made up of mock values.  Given the nature of this service, I don't think will get too complicated as the need to have different values in tests is limited.